examples/DCPS/DistributedContent/FileInfo.idl

```diff
@@ -1,10 +1,9 @@
 module DistributedContent {
-
-#pragma DCPS_DATA_TYPE "DistributedContent::FileDiff"
-#pragma DCPS_DATA_KEY "DistributedContent::FileDiff file_id"
-
   typedef sequence<octet> OctetSeq;
+
+  @topic
   struct FileDiff {
+    @key
```

Comment:
I think these `@key` annotations would be more readable if they were on the same line, since each line defines its own field.

Reply:
I don't necessarily disagree, I'm just using the style given by the spec and by usage of decorators in Python.